query-service-impl/src/test/java/org/hypertrace/core/query/service/QueryRequestBuilderUtils.java

```diff
@@ -120,6 +120,10 @@ public static Filter createInFilter(String column, List<String> values) {
     return createFilter(column, Operator.IN, createStringArrayLiteralValueExpression(values));
   }
 
+  public static Filter createBooleanInFilter(String column, List<Boolean> values) {
```

Comment:
```suggestion
  public static Filter createInFilter(String column, List<Boolean> values) {
```
This can be an overloaded method since the signature is different from existing one.

Reply:
In Java, when you use methods with generic types like List<Boolean> and List<String, the compiler can't distinguish them because they both appear as plain List without specific type information due to type erasure.

- https://www.tutorialspoint.com/java_generics/java_generics_no_overload.htm
- https://www.baeldung.com/java-type-erasure